src/modules/ROOT/pages/changelog/202303/06.json

```diff
@@ -5,18 +5,18 @@
   "added": [
     {
       "what": [
-        "splice function in workflows that can change the contents of a collection of elements by removing or replacing existing elements and/or adding new elements in place."
+        "splice function in workflows that can change the contents of an array by removing or replacing existing elements of different types like strings, numbers, etc. and/or adding new elements in place."
```

Comment:
Prefer to do those quotes around any type name such as `array` instead of array.
This is just for consistency but I won't be too naggy about it.